models/shopify.yml

```diff
@@ -251,6 +282,61 @@ models:
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - email
+            - source_relation
+    columns:
+      - name: last_fivetran_synced
+        description: "{{ doc('_fivetran_synced') }}"
+      - name: has_accepted_marketing
+        description: Whether the customer has consented to receive marketing material via email.
+      - name: created_timestamp
+        description: The date and time when the customer was created.
+      - name: default_address_id
+        description: The default address for the customer. 
+      - name: email
+        description: The unique email address of the customer.
+      - name: first_name
+        description: The customer's first name.
+      - name: customer_ids
+        description: Comma-separated list of customer IDs associated with the email.
+      - name: last_name
+        description: The customer's last name.
+      - name: lifetime_count_orders
+        description: The number of orders associated with this customer.
+      - name: phone_numbers
+        description: Comma-separated list of phone numbers associated with this email.
+      - name: account_state
+        description: The state of the customer's account with a shop.
+      - name: is_tax_exempt
+        description: Whether the customer is exempt from paying taxes on their order. If true, then taxes won't be applied to an order at checkout. If false, then taxes will be applied at checkout.
+      - name: last_updated_at
+        description: The date and time when the customer information was last updated.
+      - name: is_verified_email
+        description: Whether the customer has verified their email address.
+      - name: first_order_timestamp
+        description: The timestamp the customer completed their first order.
+      - name: most_recent_order_timestamp
+        description: The timestamp the customer completed their most recent order.
+      - name: average_order_value
+        description: The average order value for the customer.
+      - name: lifetime_total_spent
+        description: The total amount of money that the customer has spent on orders across their order history.
+      - name: lifetime_total_refunded
+        description: The total amount of money that the customer has been refunded on orders across their order history.
+      - name: lifetime_total_amount
+        description: The total amount of money (minus refunds) that the customer has spent across their order history.
+      - name: first_account_created_at
+        description: Timestamp of when the first account associated with this email was created.
+      - name: last_account_created_at
+        description: Timestamp of when the last account associated with this email was created.
+      - name: accepts_marketing_last_updated_at
+        description: When the customer consented or objected to receiving marketing material by email. 
+
```

Comment:
The `shopify__order_lines` fields need to be added it looks like (looks like they were never present there).

Reply:
i believe they're on line [383](https://github.com/fivetran/dbt_shopify/pull/45/files#diff-ca3b92d02c70e6d6d0d754cc475c032da6910c18ffeafaecda37eb7251ba2aedR383), unless you meant certain fields in particular?

Reply:
@fivetran-jamie The `variant_tax_code` and `variant_is_requiring_shipping` fields aren't documented in that model.

Reply:
Oh `variant_is_requiring_shipping` is a dead field per your comment, guess you don't have to bring that one in!

Reply:
ah gotcha i will handle this in the documentation PR